### PR TITLE
How to update innersourcecommons.org with new articles

### PR DESCRIPTION
**Pull request by rrrutledge**
View the [rendered markdown](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/docs/scripts/scripts/README.md).

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -20,6 +23,23 @@ A node script to generate markdown files required for hosting Learning Path on i
 ```
 npm ci
 node generate_learning_path_markdown.js
+```
+
+## How to update innersourcecommons.org with new articles
+
+Anytime there are _new_ articles written, you need to run the script that generates the new pages on the [innersourcecommons.org] site.
```
Comment (tsadler1988) [inline scripts/README.md:30]: There's an edge case if an existing article's title gets updated (e.g. https://github.com/InnerSourceCommons/innersourcecommons.org/pull/214) but perhaps not worth calling out.

File: scripts/README.md [hunk 2]
```diff
@@ -20,6 +23,23 @@ A node script to generate markdown files required for hosting Learning Path on i
...
+
+1. Clone the [InnerSourceCommons/InnerSourceLearningPath] and [InnerSourceCommons/innersourcecommons.org] repos.
+1. Is this the first time that articles have been written for this section in this language?
+If so, then update the ["sections" config](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/scripts/generate_learning_path_markdown.js#L37) with the language code of the articles for the appropriate section.
+Open a pull request for the change.
+1. Run **generate_learning_path_markdown.js** as described above.
+1. `cp -r learningpath/* <path-to-innersourcecommons.org-repo>/resources/learningpath/`.
```
Comment (tsadler1988) [inline scripts/README.md:37]: Missing a step - you need to update `resources/learningpath/index.md` in the innersourceommons.org repo with links to the new translation.

**Review by tsadler1988 — CHANGES_REQUESTED** [review]

File: scripts/README.md (hunk 1 above)
Comment (rrrutledge) [reply scripts/README.md:30]: Yeah I thought about that, too.

File: scripts/README.md (hunk 2 above)
Comment (rrrutledge) [reply scripts/README.md]: Updated!  Have a look.

**Review by tsadler1988 — APPROVED** [review]

**Comment by tsadler1988** [comment]
Looks good :)

**Pull request merged**